os/sdk-modifying-coreos.md

```diff
@@ -154,7 +154,7 @@ Build all of the target binary packages:
 Build an image based on the binary packages built above, including development tools:
```

Comment:
Since we're now building the prod image, `including development tools` should be dropped.

Reply:
Hi @bgilbert , thanks for catching that!